app/models/metasploit/cache/module/instance.rb

```diff
@@ -222,72 +195,61 @@ class Metasploit::Cache::Module::Instance < ActiveRecord::Base
   # through: :module_architectures
   #
 
-  # @!attribute [r] architectures
-  #   The {Metasploit::Cache::Module::Architecture architectures} supported by this module.
-  #
-  #   @return [ActiveRecord::Relation<Metasploit::Cache::Architecture>]
+  # The {Metasploit::Cache::Module::Architecture architectures} supported by this module.
   has_many :architectures, :class_name => 'Metasploit::Cache::Architecture', :through => :module_architectures
 
   #
   # through: :module_authors
   #
 
-  # @!attribute [r] authors
-  #   The names of the authors of this module.
-  #
-  #   @return [ActiveRecord::Relation<Metasploit::Cache::Author>]
+  # The names of the authors of this module.
   has_many :authors, :class_name => 'Metasploit::Cache::Author', :through => :module_authors
 
-  # @!attribute [r] email_addresses
-  #   The email addresses of the authors of this module.
-  #
-  #   @return [ActiveRecord::Relation<Metasploit::Cache::EmailAddress>]
+  # The email addresses of the authors of this module.
   has_many :email_addresses, :class_name => 'Metasploit::Cache::EmailAddress', :through => :module_authors, :uniq => true
 
   #
   # through: :module_class
   #
 
-  # @!attribute [r] rank
-  #   The rank of this module.
-  #
-  #   @return [Metasploit::Cache::Module::Rank]
+  # The rank of this module.
   has_one :rank, :class_name => 'Metasploit::Cache::Module::Rank', :through => :module_class
 
   #
   # through: :module_platforms
   #
 
-  # @!attribute [r] platforms
-  #   Platforms supported by this module.
-  #
-  #   @return [ActiveRecord::Relation<Metasploit::Cache::Module::Platform>]
+  # Platforms supported by this module.
   has_many :platforms, :class_name => 'Metasploit::Cache::Platform', :through => :module_platforms
 
   #
   # through: :module_references
   #
 
-  # @!attribute [r] references
-  #   External references to the exploit or proof-of-concept (PoC) code in this module.
-  #
-  #   @return [ActiveRecord::Relation<Metasploit::Cache::Reference>]
+  # External references to the exploit or proof-of-concept (PoC) code in this module.
   has_many :references, :class_name => 'Metasploit::Cache::Reference', :through => :module_references
 
   #
   # through: :references
   #
 
-  # @!attribute [r] authorities
-  #   Authorities across all {#references} to this module.
-  #
-  #   @return [ActiveRecord<Metasploit::Cache::Authority>]
+  # Authorities across all {#references} to this module.
   has_many :authorities, :class_name => 'Metasploit::Cache::Authority', :through => :references, :uniq => true
 
   #
   # Attributes
   #
 
+  # @!method default_action_id
```

Comment:
Is there a reason these aren't `@!attribute`s? Using `@!method` will not create a setter.